lib/WUI/nhttp/file_info.cpp

```diff
@@ -33,6 +35,63 @@ namespace nhttp::printer {
 using namespace handler;
 
 JsonResult FileInfo::DirRenderer::renderState(size_t resume_point, JsonOutput &output, FileInfo::DirState &state) const {
+    if (api == APIVersion::Octoprint)
+        return renderStateOctoprint(resume_point, output, state);
+    else
+        return renderStateV1(resume_point, output, state);
+}
```

Comment:
Nitpick: I'd prefer to always use the block brackets, even for 1-line branches, eg:

```cpp
if (...) {
    return stuff;
} else {
    return other_stuff;
}
```

That's a bit more resilient against future adding of lines.

I've already wanted to take this up during a retrospective, but the last retrospective kind of didn't really happen 😇